docs/docs/dev_docs/dapps/main.md

```diff
@@ -32,11 +32,12 @@ Explain how to write a dapp using [`aztec.js`](https://github.com/AztecProtocol/
       - How to filter for historic data in the historic block tree?
       - How to query data from any of the trees (advanced)
 
-FOR INSTRUCTIONS FOR BUILDING A WALLET, WE SHOULD WRITE DOCS [HERE](../wallets/building_a_wallet.md)
```

Comment:
@iAmMichaelConnor I removed this link because it was giving me errors with running locally. That page does not exist.